values.yaml

```diff
@@ -520,6 +520,12 @@ multiplayer:
       - path: /api/multiplayer
         port: 80
 
+  service:
```

Comment:
We have two values.yaml files unfortunately, and they need to match. Would you be able to copy this to the other one?